tests/domain_tests/TestRunner.cs

```diff
@@ -1151,6 +1263,11 @@ public static int Main()
         {{
             try
             {{
+                RuntimeData.FormatterFactory = () => 
```

Comment:
Would it be better to limit this to the `test_serialize_unserializable_object` case?

Reply:
Doesn't matter IMHO.